onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -422,19 +422,13 @@ def _where_input_wrangler(
     TorchLibOpInfo("abs", core_ops.aten_abs_complex, complex=True),
     TorchLibOpInfo("acos", core_ops.aten_acos),
     TorchLibOpInfo("acosh", core_ops.aten_acosh),
-    TorchLibOpInfo(
-        "add",
-        core_ops.aten_add,
-    ).xfail(
-        dtypes=[torch.float16],
-        reason="fixme: float16 failed, tensor-likes are not close for FullGraph mode. https://github.com/microsoft/onnxruntime/issues/15977",
-        test_class_name="TestOutputConsistencyFullGraph",
```

Comment:
Maybe we should keep the link somewhere?

Reply:
Added to the function docs